Userland/Libraries/LibWeb/CSS/Parser/Declaration.h

```diff
@@ -20,6 +20,10 @@ class Declaration {
     Declaration();
     ~Declaration();
 
+    String const& name() const { return m_name; }
```

Comment:
Does this mean the `friend` declaration above can be removed?
(Sorry, GitHub isn't letting me comment on that line specifically)

Reply:
Not yet, though that's worth doing eventually. The Parser accesses those fields when creating the Declaration, same with the other Parser-internal classes. I've made a note to go through them some time and make them all completely initialized when constructed.